src/Features/LanguageServer/Protocol/Microsoft.CodeAnalysis.LanguageServer.Protocol.csproj

```diff
@@ -14,15 +14,20 @@
   </PropertyGroup>
 
   <ItemGroup Label="Project References">
+    <ProjectReference Include="..\..\..\Compilers\CSharp\Portable\Microsoft.CodeAnalysis.CSharp.csproj" />
```

Comment:
why do you need CSharp specific dll? either you should have both CSharp and VB or none?

Reply:
For syntax facts, checking if new line.  If you know of a different appropriate helper, I can use that.  Otherwise will use this one.